examples/loader-duckdb/src/educ_uoe_lang01.parquet.sh

```diff
@@ -0,0 +1,21 @@
+export CODE="educ_uoe_lang01"
+export URL='https://ec.europa.eu/eurostat/api/dissemination/sdmx/2.1/data/'$CODE'/?format=SDMX-CSV&compressed=true&i'
+
+if [ ! -f "$TMPDIR/$CODE.csv.gz" ]; then curl "$URL" --output $TMPDIR/$CODE.csv.gz; fi
+gunzip --keep $TMPDIR/$CODE.csv.gz
+
+duckdb :memory: << EOF
+COPY (
+  SELECT *
+  FROM read_csv('$TMPDIR/$CODE.csv')
+  WHERE true
+    AND TIME_PERIOD = 2019 -- a good year in terms of data quality
+    AND OBS_VALUE > 0 -- filter out zeros
+    AND isced11 = 'ED2' -- lower secondary education
+    AND unit = 'PC' -- ignore absolute numbers, keep percentages
+    AND language != 'TOTAL' -- ignore total
+    AND length(geo) = 2 -- ignore groupings such as EU_27
+) TO '$TMPDIR/$CODE.parquet' (COMPRESSION gzip);
```

Comment:
```suggestion
) TO STDOUT (COMPRESSION gzip);
```

Can we avoid the temporary file by writing directly to stdout?

Reply:
No, this errors with `FATAL Error: fsync failed!` (even if you add `FORMAT 'parquet'` in the options). It works if you specify a csv output, but I believe the parquet writer needs random access which STDOUT doesn't provide.